google/gcloud/environment.go

```diff
@@ -0,0 +1,28 @@
+package gcloud
+
+import (
+	"github.com/pkg/errors"
+)
+
+type Environment struct {
+	Name     string    `json:"name" yaml:"name"`
+	Project  string    `json:"project" yaml:"project"`
```

Comment:
So, we did end up extracting the project?

Reply:
As discussed, but different now on the `Cluster`